docs/Dev_eXo_Addons.rst

```diff
@@ -1327,6 +1327,67 @@ To add an event listener using one of listed events above, you can add the follo
           <type>org.example.CustomEventListener</type>
         </component-plugin>
       </external-component-plugins>
+      
+      
+.. _PLFDevGuide.eXoAdd-ons.eXoOnlyOfficeConnector:
+
+==============================
+OnlyOffice connector for eXo
+==============================
+
+With :ref:`OnlyOffice connector <OnlyOffice>`, it is possible to add new document types to the
+form ``New document`` by using the :ref:`extension mechanism <PLFDevGuide.eXoAdd-ons.PortalExtension.Mechanism>`.
+For that purpose, you just need to define this external component plugin in your extension:
+
+
+   .. code:: xml
+
+      <external-component-plugins>
+		<target-component>org.exoplatform.onlyoffice.documents.NewDocumentService</target-component>
+			<component-plugin>
+				<name>documentTypePlugin</name>
+				<set-method>addTypePlugin</set-method>
+				<type>org.exoplatform.onlyoffice.documents.NewDocumentTypePlugin</type>
+				<description>Add new document types to create</description>
+			<init-params>
+			<object-param>
+				<name>document-types-configuration</name>
+				<object type="org.exoplatform.onlyoffice.documents.NewDocumentService$NewDocumentTypesConfig">
+					<field name="types">
+						<collection type="java.util.ArrayList">
+						<value>
+				<object type="org.exoplatform.onlyoffice.documents.NewDocumentType">
+					<field name="path">
+						<string>classpath:files/template.docx</string>
+					</field>
+					<field name="label">
+						<string>MicrosoftOfficeDocument</string>
+					</field>
+					<field name="mimeType">
+						<string>application/vnd.openxmlformats-officedocument.wordprocessingml.document</string>
+					</field>
+				</object>
+					</value>
+			</collection>
+			</field>
+				</object>
+			</object-param>
+			</init-params>
+			</component-plugin>
+		</external-component-plugins>  
+		
+Each new document templated to add should be defined by this object ``org.exoplatform.onlyoffice.documents.NewDocumentType``
+with these parameters:
+
+-  ``path``: Defines the path to the created document's template. 
+-  ``label``: The template's label. 
+-  ``mimeType``: The document's mimetype.
+
+.. tip:: -  The ``path`` should be defined as it is made in eXo Platform components i.e. when it is preceeded
+            by **classpath** it means the template should be found on jars and when it is preceeded by 
+            **war:** it means the template should be found under webapps.
+         -  The parameter ``label`` is the traduction key for the label to be displayed in the ``New Document``
+            form. To define it, you should configure the parameter ``UINewDocumentForm.label.option``.
```

Comment:
Why not keeping these explanations in the first list directly ?

Reply:
OK

Reply:
done